fwutil/lib.py

```diff
@@ -903,6 +903,12 @@ def is_first_auto_update(self, boot):
                 return False
         for status_file in glob.glob(os.path.join(FIRMWARE_AU_STATUS_DIR, FW_AU_STATUS_FILE)):
             if status_file is not None:
+                data = self.read_au_status_file_if_exists(FW_AU_STATUS_FILE_PATH)
+                if data is not None:
+                    if boot is "none" or boot in data:
+                        click.echo("Allow firmware auto-update {} again on top of the previous {} reboot".format(boot, boot_type))
```

Comment:
Has this been manually tested? Do we have any unit tests for this?

`boot_type` is not defined in this latest change as far as I can tell.